cmd/kubetoken/kubetoken.go

```diff
@@ -53,17 +67,26 @@ You can find more information on 'ExecCredential' and 'client side authenticatio
 			return errors.ErrContextIsNotOktetoCluster
 		}
 
-		c, err := okteto.NewKubeTokenClient(octx.Name, octx.Token, octx.Namespace)
+		c, err := okteto.NewOktetoClient()
 		if err != nil {
 			return fmt.Errorf("failed to initialize the kubetoken client: %w", err)
 		}
 
-		out, err := c.GetKubeToken()
+		out, err := c.Kubetoken().GetKubeToken(octx.Name, octx.Namespace)
 		if err != nil {
 			return fmt.Errorf("failed to get the kubetoken: %w", err)
 		}
 
-		cmd.Print(out)
+		serializer := &Serializer{
+			KubeToken: out,
+		}
+
+		jsonStr, err := serializer.ToJson()
+		if err != nil {
+			return fmt.Errorf("failed to marshal KubeTokenResponse: %w", err)
```

Comment:
Is this logged somewhere?

Reply:
unfortunatelty not and we don't have a log able to do it as of today :(